cfgov/paying_for_college/jinja2/paying-for-college/college-cost-blocks/03-grants-and-scholarships.html

```diff
@@ -0,0 +1,168 @@
+<section class="college-costs_tool-section" data-tool-section="grants-scholarships">
+    <h2>Grants and scholarships</h2>
+    <p>You do not have to pay back grants and scholarships as long as you meet their requirements. Please enter the grants and scholarships from your financial aid offer as well as any others you have been awarded.</p>
+    {{ notification.render(
+            'default',
+            true,
+            '',
+            'Not sure where your grants and scholarships are listed? One way to identify grants and scholarships on your offer is to use the Hechinger Report’s OfferLetterDECODER, which color codes the different types of funding in offers uploaded as PDFs. (This links to a third-party resource. We do not endorse the third-party or guarantee the accuracy of this third-party information. There may be other resources that also serve your needs.)'
+    ) }} 
+    <div class="grants-and-scholarships_content-section financial-items">
+        <div class="block block__mid">
+            <h3 class="header__border-bottom">Grants</h3>
```

Comment:
Modifiers should technically modify the base block, so you'd include the base block and the modifier with it. So you'd have `header header__border-bottom`, like you have `block block__mid` in the div above this line.

Reply:
Modifiers can also modify the element.

Reply:
Yes, there's no base `header` class, so I updated to a generic `u-border-bottom` class in https://github.com/cfpb/cfgov-refresh/pull/5742/commits/9237cac212c52dfd381dd65958ae6f2af5f8570c.